beacon_node/network/src/sync/range_sync/chain.rs

```diff
@@ -891,7 +920,7 @@ impl<T: BeaconChainTypes> SyncingChain<T> {
                 .network_globals()
                 .peers
                 .read()
-                .synced_peers()
+                .synced_peers_for_epoch(batch_id, &self.peers)
```

Comment:
The existing code already assumes `batch_id` == `epoch`, but i think we might want to start moving away from it, both for clarity and safety in case we want to change the batch size in the future. What about we get the epoch from the batch `start_slot`?